src/commonmark/en/content/user/scheduling.md

```diff
@@ -98,6 +98,15 @@ Further configuring a job:
 
 The following section describes the various job types.
 
+### Disable Inactive Users
+
+<!--DHIS2-SECTION-ID:scheduling_disable_inactive_users-->
+Users that have not been active - not logged in - for a number of months can automatically be disabled.
+Select the number of months users at least have to be inactive as job parameter. 
```

Comment:
This sentence is a little awkward. Suggestion:
"Select the number of inactive months as the job parameter."

The following sentence goes on to explain the rest. (Though it sounds clearer to me if you change "when running the job" to simply "by the job").

Reply:
done.